src/openms/source/CHEMISTRY/EnzymaticDigestion.cpp

```diff
@@ -86,6 +87,7 @@ namespace OpenMS
   void EnzymaticDigestion::setEnzyme(const String enzyme_name)
   {
     enzyme_ = *EnzymesDB::getInstance()->getEnzyme(enzyme_name);
+    if (enzyme_name == "unspecific cleavage") { missed_cleavages_ = std::numeric_limits<Size>::max(); }
```

Comment:
this is a bad place to put this.
what if the user calls 'setMissedCleavages()' afterwards? 
We need to come up with a better design here. Probably inside 'digest()'.